src/HttpClient/ClientOptions.php

```diff
@@ -0,0 +1,111 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\Externals\HttpClient;
+
+use EoneoPay\Externals\HttpClient\Interfaces\ClientOptionsInterface;
+use GuzzleHttp\RequestOptions;
+
+class ClientOptions implements ClientOptionsInterface
+{
+    /**
+     * @var float The number of seconds the HTTP client will wait to connect.
+     */
+    private $connectTimeout = 5;
```

Comment:
Given our infrastructure, a connect timeout should be really low - 1s is appropriate